xdmod-appkernels.spec.in

```diff
@@ -51,9 +51,14 @@ rm -rf $RPM_BUILD_ROOT
 %{_libdir}/xdmod/
 %{_datadir}/xdmod/
 %{_docdir}/%{name}-%{version}__PRERELEASE__/
+%{_docdir}/%{name}-%{version}/
```

Comment:
This line should be unnecessary since the line above it is intended to include the documentation files.  If the pre-release versioning is broken please let me know.